intercom-java/src/main/java/io/intercom/api/Company.java

```diff
@@ -353,6 +392,9 @@ public boolean equals(Object o) {
         if (untag != null ? !untag.equals(company.untag) : company.untag != null) return false;
         //noinspection RedundantIfStatement
         if (userCount != null ? !userCount.equals(company.userCount) : company.userCount != null) return false;
+        if (size != company.size) return false;
+        if (website != null ? !website.equals(company.website) : company.website != null) return false;
```

Comment:
This is quite complicated to read, you want to make sure that the current website and company website aren't equal, right? Is there any way you could export the equals checks to a separate method that returns the boolean?

Reply:
Yeah I can see that it is essentially checking for the possibly inequality between the 2 objects. I was following the standard format that exists for string checking and it is utilised in several places. Could definitely use a refactor though 👍 

Do you see any issues with `if (!Objects.equals(website, company.website)) return false;` https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#equals(java.lang.Object,%20java.lang.Object)

I can try implement that as it seems built but seems to require Java 1.7

Reply:
That's definitely a lot easier to read. What version of Java is this on? Why isn't it up higher?

That would definitely be nice, but I can understand that you're following the format. As well though, maintainability and readability are definitely important 😄 

This definitely isn't a breaking change though.

Reply:
Don't think we have specified a version of Java we support but created this issue https://github.com/intercom/intercom-java/issues/214 to track the possibility of simplifying this

Reply:
Sorry Tim, I missed this. I think you're definitely right we need to look into simplifying it.